### PR TITLE
[NEW] Project: Create a new page about task dependencies

### PR DESCRIPTION
**Pull request by guga-odoo**

### CODE REVIEW

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @guga-odoo thank you for your work! 
- I think we could document this feature for the 17.0 version (i.e., rebase the branch) and make the few required changes for 18.0 on the fw-port branch (we can do it together!)
- I suggested quite a few changes structure-wise and I’m not sure I explained them clearly. If anything is confusing, please don’t hesitate to reach out—it’ll be easier to clarify in person! :sweat_smile:

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+.. image:: task_dependencies/task-dependency.png
+   :alt: Task dependency
+
+Remove dependencies
+===================
+
+- From the task form: go to the **Blocked by** tab and click the :icon:`fa-times` button.
+- From Gantt view: click the red :guilabel:`X` button that appears at the center of the arrow when
+  you hover your mouse over it.
```
[the remark this answers is not in the document]
Comment (guga-odoo) [reply content/applications/services/project/tasks/task_dependencies.rst:52]: AJU told me the interface will soon change and the arrow buttons will disappear. I'll update this page once the updated interface is live.

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+=================
+Task dependencies
+=================
+
+Odoo Project allows you to break down projects into tasks and establish relationships between those
+tasks to determine the order in which they are executed. Task dependencies ensure that certain tasks
+begin only after the preceding tasks are completed.
+
+To enable task dependencies in Projects, go to :menuselection:`Project --> Configuration -->
```
Comment (auva-odoo) [inline content/applications/services/project/tasks/task_dependencies.rst:9]:
```suggestion
To enable task dependencies in projects, go to :menuselection:`Project --> Configuration -->
```

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Settings`, enable :guilabel:`Task Dependencies`, and click the :icon:`fa-cloud-upload`
+(:guilabel:`save`) icon.
```
Comment (auva-odoo) [inline content/applications/services/project/tasks/task_dependencies.rst:11]:
```suggestion
Settings`, enable :guilabel:`Task Dependencies`, and click :guilabel:`Save`).
```

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Set task dependencies
+=====================
+
+Task dependencies can be created from the task form or the project's Gantt view by linking the
+successor task (i.e., the task blocked by other tasks) to its predecessor task(s) (i.e., the tasks
+blocking the successor task).
+
+To create task dependencies from the task form, access the desired task and, in the
+:guilabel:`:guilabel:`Blocked by` tab, click :guilabel:`Add a line`. To access successor tasks from
+the predecessor task, click the :guilabel:`Blocking Tasks` smart button.
```
Comment (auva-odoo) [inline content/applications/services/project/tasks/task_dependencies.rst:22]:
```suggestion
:guilabel:`Blocked by` tab, click :guilabel:`Add a line`. Click :guilabel:`View` to access the predecessor task. To access the successor tasks from
the predecessor task, click the :guilabel:`Blocking Tasks` smart button.
```
(becomes "Blocked Tasks" in 18.0)

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+To create a task dependency from the Gantt view, hover your mouse over the predecessor task, then
+click one of the dots that appear around it. Drag and drop the dot onto the successor task. An arrow
+will appear, indicating the dependency from the predecessor task to the successor.
```
Comment (auva-odoo) [inline content/applications/services/project/tasks/task_dependencies.rst:26]:
```suggestion
click one of the dots that appear around it. Drag and drop the dot onto the successor task. An arrow appears, indicating the dependency from the predecessor task to the successor.
```

File: content/applications/services/project/tasks/task_dependencies.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Odoo automatically manages task progress based on their dependency. Successor tasks are assigned the
+:guilabel:`Waiting` status and cannot be moved to :guilabel:`In Progress` until their predecessor
+task(s) are marked as :guilabel:`Approved`, :guilabel:`Cancelled` or :guilabel:`Done`.
+
+Remove dependencies
+===================
+
+- From the task form: go to the **Blocked by** tab and click the :icon:`fa-times` button.
+- From Gantt view: click the red :guilabel:`X` button that appears at the center of the arrow when
+  you hover your mouse over it.
```
Comment (auva-odoo) [inline content/applications/services/project/tasks/task_dependencies.rst:40]:
```suggestion
To remove a task dependency, proceed as follows:
- From the task form, go to the **Blocked by** tab and click the :icon:`fa-times` button.
- From Gantt view, click the red :guilabel:`X` button that appears at the center of the arrow when
  you hover your mouse over it.
```

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @guga-odoo, I have a few more comments, but I suggest we rebase your branch to 17.0 first (we'll do it together, let me know when you're ready!).

**Comment by guga-odoo** [comment]
Created new branch in 17.0 https://github.com/odoo/documentation/pull/11657

**Pull request closed without merging**